presets/4.4/tune/karate/tiny_karate.txt

```diff
@@ -108,3 +108,11 @@ set tpa_breakpoint = 1250
     set dyn_idle_min_rpm = 60
     set dyn_idle_p_gain = 35
 #$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): Quaddiction LED channel mapping
+    #$ INCLUDE: presets/4.4/LEDS/quaddition TT leds.txt 
```

Comment:
The filename has whitespaces in it. You must not use whitespaces, so replacing them with underscores in the filename fixes the problem.

Reply:
ohh yeah sorry I'll fix it up